pkg/cmd/workspace/util/repository_wizard.go

```diff
@@ -134,7 +134,12 @@ func getRepositoryFromWizard(config RepositoryWizardConfig) (*apiclient.GitRepos
 			disablePagination = true
 		} else {
 			// Check if we have reached the end of the list
-			disablePagination = int32(curPageItemsNum) < perPage
+			// Edge Case for github, gitlab and gitea providers since we manually append "personal" namespace info on the first page
+			if providerId == "github" || providerId == "gitlab" || providerId == "gitea" && page == 1 {
```

Comment:
Let's create a function for this so it's more clear why does this edge case exist.

e.g. `func providerAppendsPersonalNamespace(providerId string) { return providerId == "github" || providerId == "gitlab" || providerId == "gitea" }`

Reply:
done!